pkg/combustion/templates/99-cleanup-combustion.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+rm -r /combustion
+
+if test -d /artefacts; then
+		rm -r /artefacts
```

Comment:
Can we improve the indentation here?

Reply:
I have no idea why it does that, it looked like 1 tab in Goland but I just fixed it

Reply:
hint: you should be able to configure what a tab does in the settings